core/ecschema-metadata/src/Editor/Editor.ts

```diff
@@ -910,7 +910,7 @@ export namespace Editors {
       if (koqProps.name === undefined) return { errorMessage: `No name was supplied within props.` };
       const newKoQ = (await schema.createKindOfQuantity(koqProps.name)) as MutableKindOfQuantity;
       if (newKoQ === undefined) {
-        return { errorMessage: `Failed to create class ${name} in schema ${schemaKey.toString(true)}.` };
+        return { errorMessage: `Failed to create class ${koqProps.name} in schema ${schemaKey.toString(true)}.` };
```

Comment:
@calebmshafer This probably should have been an error before, but it was actually using a global `name` var from lib.dom.d.ts before.  That name var has now been deprecated, so I had to guess at what this was supposed to be...